cypress/e2e/api/testsApi.cy.js

```diff
@@ -0,0 +1,135 @@
+describe('Validate the API requests', () => {
+  const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`;
+
+  context('Pagination and limits', () => {
+    it('Get customers successfully', () => {
+      cy.request({
+        method: 'GET',
+        url: CUSTOMERS_API_URL,
+      }).then(({ status }) => {
+        expect(status).to.equal(200);
+      });
+    });
+
+    it('Get the second page', () => {
+      cy.request({
+        method: 'GET',
+        url: `${CUSTOMERS_API_URL}?page=2`,
+      }).then(({ status }) => {
+        expect(status).to.equal(200);
+      });
+    });
+
+    it('Change the limits of customers per page', () => {
+      cy.request({
+        method: 'GET',
+        url: `${CUSTOMERS_API_URL}?page=1&limit=20`,
+      }).then(({ status, body }) => {
+        expect(status).to.equal(200);
+        expect(body.pageInfo.totalPages).to.equal(3);
+      });
+    });
+  });
+  context('Filters', () => {
+    const sizes = ['Small', 'Medium', 'Enterprise', 'Large Enterprise', 'Very Large Enterprise'];
+    const industries = ['Logistics', 'Retail', 'Technology', 'HR', 'Finance'];
```

Comment:
```suggestion
    const industries = ['Logistics', 'Retail', 'Technology', 'HR', 'Finance'];

```

É uma boa prática e convenção separar a definição de variáveis de outras expressões por uma linha em branco.

Isso ajuda na legibilidade do código.